File: Data Structures & Algorithms/multiply-strings/submission-3.py
class Solution:
    def multiply(self, num1: str, num2: str) -> str:
        n = int(num1);
        nn = int(num2)
        return (str(n * nn))

"""
    string multiply(string num1, string num2) {
            if((num1.size() >= 1 && num1.size() <= 200) && (num1.size() >= 1 && num1.size() <= 200))
            {
                size_t n = stoi(num1);
                size_t nn = stoi(num2);
                return(to_string(n * nn));
            }
            return string("");
            
    }
"""

File: Data Structures & Algorithms/two-integer-sum-ii/submission-3.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& n, int t)
    {
        for(int i = 0; i < n.size(); i++)
        {
            const int e = n[i];
            for(int j = i+1; j < n.size(); j++)
            {
                if((e + n[j]) == t)
                    return {i+1, j+1};
            }
        }
    }
};
